### DISCUSSION TITLE
Update Patch Notes.

### DISCUSSION THREAD

**Discussion by neeestle** [seerge/g-helper#2720] [Ideas]
I'd like to see what changes everytime there's a new update with patch notes, if possible. 

Well technically my laptop is on **life support**, and g-helper is the only thing that keeps my laptop alive [thank you!] (due to undefined BSOD issues), i'd like to know if future updates will depecrate older features that can cause whatever magic that was working before to be lost in time.

**Comment by seerge** [comment]
@neeestle you can read what's new on [the releases page](https://github.com/seerge/g-helper/releases) or by clicking on version label in the app

**Reply by neeestle** [reply]
i have a hard time finding releases page, maybe highlight it on the first page (i mean on readme.md)/before updating from app that can act like a notices.
